src/argbash-1to2.m4

```diff
@@ -19,36 +19,37 @@ cleanup()
 	test "${#_files_to_clean[*]}" != 0 && rm -f "${_files_to_clean[@]}"
 }
 
-do_stuff ()
+do_stuff()
 {
 	# SCRIPT_DIR is likely also a default, but maybe not - it may have been set explicitly
-	grep -q '\${\?SCRIPT_DIR' -- "$infname" && echo "You probably use a variable 'SCRIPT_DIR' in your script. It may be that you should rename it to 'script_dir', but this is not certain :-(" >&2
+	grep -q '\${\?SCRIPT_DIR' -- "${infname}" && echo "You probably use a variable 'SCRIPT_DIR' in your script. It may be that you should rename it to 'script_dir', but this is not certain :-(" >&2
 	# We match $_ARG_FOO as well as ${ARG_FOO...
 	# and _ARGS_FOO
-	sed 's/\(\${\?_ARGS\?_\w\+\)/\L\1\l/g' "$infname"
+	sed 's/\(\${\?_ARGS\?_\w\+\)/\L\1\l/g' "${infname}"
 }
 
-outfname="$_arg_output"
-test "${#infname[@]}" -gt 1 && test -n "$outfname" && die "You have specified more than one (${#infname[@]}) input filenames, so you probably want to modify the corresponding files in-place. In order to do so, you can't specify an output filename, even '-' does make no sense (currently: '$outfname')"
+outfname="${_arg_output}"
+test "${#infname[@]}" -gt 1 && test -n "${outfname}" && die "You have specified more than one (${#infname[@]}) input filenames, so you probably want to modify the corresponding files in-place. In order to do so, you can't specify an output filename, even '-' does make no sense (currently: '${outfname}')"
 
 trap cleanup EXIT
-for infname in "${_arg_input[@]}"
-do
-	test -f "$infname" || { echo "The input parameter has to be a file (got: '$infname')" >&2; exit 1; }
-
-	test -n "$_arg_output" || outfname="$infname"
-	if test "$outfname" = '-'
-	then
+for infname in "${_arg_input[@]}"; do
+	test -f "${infname}" || {
+		echo "The input parameter has to be a file (got: '${infname}')" >&2
+		exit 1
+	}
```

Comment:
What about using `die` here? I don't have an idea why it wasn't used there before.
```suggestion
	test -f "${infname}" || die "The input parameter has to be a file (got: '${infname}')"
```